src/components/map/MapList.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import { useQuery } from "react-query";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import styled, { ThemeProvider } from "styled-components";
+
+import Map from "./Map";
+import ErrorPage from "../error/ErrorPage";
+import theme from "../../styles/theme";
+import StyledButton from "../common/CommonStyle";
+import ERROR_MESSAGE from "../../constants";
+
+const MapList = () => {
+  const navigate = useNavigate();
+
+  const [mapList, setMapList] = useState([]);
+  const [errorMsg, setErrorMsg] = useState("");
+  const { user } = useSelector(state => state.user);
+  const userId = user.id;
+
+  const mapListData = () => {
+    return axios.get(`/user/${userId}/maps`);
+  };
+
+  const onSuccess = ({ data }) => {
+    if (data.maps) {
+      setMapList(data.maps);
+    }
+
+    if (data.error) {
+      if (data.error.status === 400) {
+        setErrorMsg(ERROR_MESSAGE.BAD_REQUEST);
+      }
+
+      if (data.error.status === 500) {
+        setErrorMsg(ERROR_MESSAGE.SERVER_USTABLE);
+      }
+    }
+  };
+
+  useQuery("mapList", mapListData, {
+    enabled: !!userId,
```

Comment:
```suggestion
    enabled: userId,
```

혹시 `!!userId` 말고 `userId` 이렇게 사용할 수 있을까요-?

Reply:
userId가 string값인데, 그것을 boolean type으로 변경하기 위해 쌍느낌표를 사용했습니당  
[관련자료 👀  공유드립니다~](https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript)

Reply:
저는 이 부분에 boolean 값이 들어간다는 것을 명시적으로 보여주는 역할이 필요할 것 같아서 `!!` 를 함께 사용해준 것이 좋아보이는 것 같아요~